src/psyclone/psyGen.py

```diff
@@ -5335,6 +5393,58 @@ def else_body(self):
             return self._children[2]
         return None
 
+    @staticmethod
+    def create(if_condition, if_body, else_body=None):
+        '''Create an IfBlock instance given valid instances of an
+        if_condition, an if_body and an optional else_body.
+
+        :param if_condition: the PSyIR node containing the if \
+            condition of the if block.
+        :type if_condition: :py:class:`psyclone.psyGen.Node`
+        :param if_body: the PSyIR node containing the if body of \
+            the if block.
+        :type if_body: :py:class:`psyclone.psyGen.Schedule`
+        :param else_body: PSyIR node containing the else body of the \
+            if block of None if there is no else body (defaults to None).
+        :type else_body: :py:class:`psyclone.psyGen.Schedule` or \
+            NoneType
+
+        :returns: an IfBlock instance.
+        :rtype: :py:class:`psyclone.psyGen.IfBlock`
+
```

Comment:
docstring `:raises xxx:` missing

Reply:
Updated this and all other create methods.